assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -799,5 +799,53 @@ describe( 'core/modules modules', () => {
 				expect( isConnected ).toBeUndefined();
 			} );
 		} );
+
+		describe( 'getModuleFeatures', () => {
+			it( 'returns undefined when no modules are loaded', async () => {
+				fetchMock.getOnce(
+					/^\/google-site-kit\/v1\/core\/modules\/data\/list/,
+					{ body: FIXTURES, status: 200 }
+				);
+				const slug = 'analytics';
+				const featuresLoaded = registry.select( STORE_NAME ).getModuleFeatures( slug );
+
+				// The modules will be undefined whilst loading.
+				expect( featuresLoaded ).toBeUndefined();
+			} );
+
+			it( `returns features when modules are loaded`, async () => {
+				fetchMock.getOnce(
+					/^\/google-site-kit\/v1\/core\/modules\/data\/list/,
+					{ body: FIXTURES, status: 200 }
+				);
```

Comment:
Please receive modules directly rather than using a fetch mock for tests that don't need an unloaded state or test the resolver.